File: app/src/main/java/com/tosh/poolandroid/View/AppIntro/AppIntroActivity.java
package com.tosh.poolandroid.View.AppIntro;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.os.Bundle;

import com.tosh.poolandroid.R;

public class AppIntroActivity extends AppCompatActivity {

    ViewPager viewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_app_intro);

        viewPager = findViewById(R.id.viewPager);

        IntroAdapter adapter = new IntroAdapter(getSupportFragmentManager());
        viewPager.setAdapter(adapter);
    }
}

File: app/src/main/java/com/tosh/poolandroid/view/appIntro/AppIntroActivity.kt
package com.tosh.poolandroid.view.appIntro

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.tosh.poolandroid.R

class AppIntroActivity : AppCompatActivity() {

    internal lateinit var viewPager: ViewPager

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_app_intro)

        viewPager = findViewById(R.id.viewPager)

        val adapter = IntroAdapter(supportFragmentManager)
        viewPager.adapter = adapter
    }
}